### PR TITLE
Fix four comma typos

### PR DESCRIPTION
**Pull request by a-a-GiTHuB-a-a**
The documentation had a few typos that messed up syntax highlighting. This PR fixes that. (that's literally it. four missing commas.)

### CODE REVIEW

**Review by jsumners — DISMISSED** [review]
Looks good to me.

File: README.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ import Fastify from 'fastify'
 const fastify = Fastify()
 await fastify.register(import('@fastify/throttle'), {
   bytesPerSecond: 1024 * 1024 // 1MB/s
```
Comment (jsumners) [inline README.md:23]: Shouldn't there be a comma on this line as well?

Comment (a-a-GiTHuB-a-a) [reply README.md]: You're right—I missed a spot. I just fixed it.

**Review by Uzlopak — APPROVED** [review]
LGTM

**Pull request merged**